Automattic/wp-calypso
# Signup: Add testimonials and background decoration

#### Changes proposed in this Pull Request

* Experiment with balancing the two-columns layout using background elements and the addition of a testimonial in the lower right.
* See also #27862 

Before: 

<img width="1440" alt="46971801-fcca9800-d08a-11e8-940c-1bf5b9a85b9e" src="https://user-images.githubusercontent.com/2124984/47171682-52938000-d2d7-11e8-86da-dfb2ac9cef77.png">

After:

<img width="1440" alt="screen shot 2018-10-19 at 11 57 31 am" src="https://user-images.githubusercontent.com/2124984/47229653-45899600-d396-11e8-9a87-05cf93bbc27a.png">

#### Testing instructions

* Switch to this PR and navigate to `/start`
* Check `/about` and `/user` (user-first signup when logged out, may need to check for the A/B test to see it) to see visual changes.
* Note visual changes and testimonial addition.
